index.js

```diff
@@ -0,0 +1,38 @@
+const yaml = require('js-yaml');
+
+module.exports = robot => {
+    robot.on('pull_request.opened', async context => {
+        let config;
+        const files = await context.github.pullRequests.getFiles(context.issue());
+        const docs = files.data.find(function (file) {
+            if (file.filename === 'README.md' || file.filename.includes('docs/')) {
```

Comment:
Oh, and some old farts that I know still write `README` (no extension) or `README.txt`.